### PR TITLE
Fixes for live widget pr

### PR DESCRIPTION
**Pull request by AndriiDovhopolov-okta**
## Description:
- **What's changed?**  Fixes for leftovers comments
- **Is this PR related to a Monolith release?** No

### CODE REVIEW

**Comment by AndriiDovhopolov-okta** [comment]
https://github.com/okta/okta-developer-docs/pull/1931#discussion_r590618096

**Comment by AndriiDovhopolov-okta** [comment]
https://github.com/okta/okta-developer-docs/pull/1931#discussion_r597844534

**Comment by AndriiDovhopolov-okta** [comment]
https://github.com/okta/okta-developer-docs/pull/1931#discussion_r597845148

File: packages/@okta/vuepress-theme-prose/components/LiveWidgetOktaWidget.vue [hunk 1]
```diff
@@ -63,7 +61,13 @@ export default {
           }
         });
       this.widget.renderEl({ el: '#widget-container' });
+    },
+    destroyWidget(){
+       if (this.widget) {
+      this.widget.remove();
     }
   }
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/components/LiveWidgetOktaWidget.vue:69]: Can you auto-format or manually update the indentations here?

**Review by shawyu-okta — APPROVED** [review]
Small fix but otherwise lgtm

File: packages/@okta/vuepress-theme-prose/components/LiveWidgetOktaWidget.vue (hunk 1 above)
Comment (AndriiDovhopolov-okta) [reply packages/@okta/vuepress-theme-prose/components/LiveWidgetOktaWidget.vue:69]: I will kill my VS Code with fire. Thanks.

**Comment by shawyu-okta** [comment]
Can you please try to address a few items?

# The title of the page should match the existing one. Right now it's "LiveWidget" and it should say "Developer Sign In Customization"
# The "not-allowed" cursor style feel unnecessary for when hovering the currently selected tab. See the existing behavior.
# Editing the JS tab causes the refreshed widget to steal the cursor focus from the edit field (the current one does not do this)

**Comment by shawyu-okta** [comment]
Deployed page: https://6058c7017b8ef000befde8a3--reverent-murdock-829d24.netlify.app/live-widget/

**Review by shawyu-okta — APPROVED** [review]

**Pull request merged**